hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -5,13 +5,28 @@
 import React from 'react';
 import Tab from 'react-bootstrap/Tab';
 import Tabs from 'react-bootstrap/Tabs';
+import {useHistory, useParams} from 'react-router-dom';
 import ActionSettingsTab from './settings/ActionSettingsTab';
 import SignalSettingsTab from './settings/SignalSettingsTab';
 
 export default function Settings() {
+  const {tab} = useParams();
+  const history = useHistory();
+  if (
+    tab === undefined ||
+    !tab ||
+    (tab !== 'signals' && tab !== 'pipeline' && tab !== 'actions')
```

Comment:
How difficult would it be to read this tab list from somewhere because I already know its going to change (Mengyang has a ThreatExchange Tab coming)

Reply:
good call! done in a subsequent PR.